Streaming causes error message to not print on zsh

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### What happened?

I used the streaming feature in powershell on Windows and it works fine and displays error messages, however testing this on macOS zsh with any command that would result in an error will not show the error message. It would only show the error status (an x). Tested on two macOS zsh devices, with multiple timeouts (0, 1, 100, 1000). Have not yet tried looking at the source code for why this may happen, great feature though!

### Theme

Using the default theme exported with `oh-my-posh config export --output ~/.mytheme.omp.json` and added streaming with 100ms `"streaming": 100,`.

### What OS are you seeing the problem on?

macOS

### Which shell are you using?

zsh

### Log output

```Shell
Version: 29.6.1

Shell: shell (5.9)

Prompt:

 dada   ~                                                              in shell at 18:48:02

Segments:

ConsoleTitle(true)                        -   0 ms
Session(true)                             -   3 ms
Path(true)                                -   0 ms
Git(false)                                -   0 ms
Root(false)                               -   0 ms
Status(true)                              -   0 ms
Node(false)                               -   0 ms
Go(false)                                 -   0 ms
Python(false)                             -   0 ms
Shell(true)                               -   0 ms
Time(true)                                -   0 ms

Run duration: 17.87125ms

Cache path: /Users/dada/.cache/oh-my-posh

Config path: /Users/dada/.cache/oh-my-posh/mytheme.omp.json

Logs:

[DEBUG] 18:48:02.808 debug.go:39 → logging enabled, raw mode: true
[DEBUG] 18:48:02.808 store.go:]:141 → (session) key not found: prompt_count_cache
[TRACE] 18:48:02.808 store.go:](session prompt_count_cache) - 11.125µs
[TRACE] 18:48:02.808 terminal.go:setPromptCount() - 15.5µs
[TRACE] 18:48:02.808 terminal.go:GOOS() - 208ns
[DEBUG] 18:48:02.808 terminal.go:setPwd:102 → /Users/dada
[TRACE] 18:48:02.808 terminal.go:setPwd() - 24.75µs
[TRACE] 18:48:02.808 terminal.go:Init() - 202.667µs
[DEBUG] 18:48:02.808 home.go:12 → /Users/dada
[TRACE] 18:48:02.808 path.go:Path() - 14.791µs
[DEBUG] 18:48:02.808 store.go:init:94 → (device) loading is_wsl
[DEBUG] 18:48:02.808 store.go:init:94 → (device) loading https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/tokyonight_storm.omp.json.data
[DEBUG] 18:48:02.808 store.go:init:94 → (device) loading environment_platform
[DEBUG] 18:48:02.808 store.go:init:94 → (device) loading DSC_CONFIGURATION
[DEBUG] 18:48:02.808 store.go:init:94 → (device) loading DSC_SHELL
[DEBUG] 18:48:02.808 store.go:init:94 → (device) loading INITVERSIONZSH
[DEBUG] 18:48:02.808 store.go:init:94 → (device) loading NUAUTOLOADDIR
[DEBUG] 18:48:02.808 store.go:init:94 → (device) loading https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/catppuccin.omp.json.data
[DEBUG] 18:48:02.808 store.go:init:94 → (device) loading https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/tokyonight_storm.omp.json.etag
[DEBUG] 18:48:02.808 store.go:init:94 → (device) loading https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/catppuccin.omp.json.etag
[DEBUG] 18:48:02.808 store.go:init:94 → (device) loading INITVERSIONNU
[DEBUG] 18:48:02.808 store.go:init:94 → (device) loading https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/chips.omp.json.etag
[DEBUG] 18:48:02.808 store.go:init:94 → (device) loading https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/chips.omp.json.data
[TRACE] 18:48:02.808 store.go:init(device omp.cache) - 221.291µs
[TRACE] 18:48:02.808 init.go:SessionID() - 500ns
[TRACE] 18:48:02.808 path.go:Path() - 84ns
[DEBUG] 18:48:02.809 store.go:init:94 → (session) loading prompt_count_cache
[DEBUG] 18:48:02.809 store.go:init:94 → (session) loading CONFIG_SOURCE
[DEBUG] 18:48:02.809 store.go:init:94 → (session) loading CONFIG
[DEBUG] 18:48:02.809 store.go:init:94 → (session) loading template_cache
[DEBUG] 18:48:02.809 store.go:init:94 → (session) loading toggle_cache
[TRACE] 18:48:02.809 store.go:init(session zsh.adf755df-c8c5-4d38-99c8-f50fec46712e.omp.cache) - 110.875µs
[DEBUG] 18:48:02.809 store.go:]:141 → (device) key not found: reload
[TRACE] 18:48:02.809 store.go:](device reload) - 343.959µs
[DEBUG] 18:48:02.809 store.go:]:154 → (session) found entry: CONFIG - /gIHfwMBAQZDb25maWcB/4AAASABB1BhbGV0dGUB/4IAAQtEZWJ1Z1Byb21wdAH/hAABA1ZhcgH/jgABCFBhbGV0dGVzAf+QAAEJVmFsaWRMaW5lAf+EAAEPU2Vjb25kYXJ5UHJvbXB0Af+EAAEPVHJhbnNpZW50UHJvbXB0Af+EAAEJRXJyb3JMaW5lAf+EAAEETWFwcwH/lAABB1VwZ3JhZGUB/5gAAQdFeHRlbmRzAQwAAQtBY2NlbnRDb2xvcgEMAAEUQ29uc29sZVRpdGxlVGVtcGxhdGUBDAABA1BXRAEMAAEGU291cmNlAQwAAQZGb3JtYXQBDAABElRlcm1pbmFsQmFja2dyb3VuZAEMAAEOVG9vbFRpcHNBY3Rpb24BDAABBkJsb2NrcwH/ngABBUN5Y2xlAf+iAAENSVRlcm1GZWF0dXJlcwH/pAABCFRvb2x0aXBzAf+cAAEHVmVyc2lvbgEEAAENTWlncmF0ZUdseXBocwECAAEFQXN5bmMBAgABEFNoZWxsSW50ZWdyYXRpb24BAgABCkZpbmFsU3BhY2UBAgABDVVwZ3JhZGVOb3RpY2UBAgABDlBhdGNoUHdzaEJsZWVkAQIAAQtBdXRvVXBncmFkZQECAAEXRW5hYmxlQ3Vyc29yUG9zaXRpb25pbmcBAgABCVN0cmVhbWluZwEEAAAAF/+BBAEBB1BhbGV0dGUB/4IAAQwBDAAA/gIx/4MDAQEHU2VnbWVudAH/hAABJAEHT3B0aW9ucwH/hgABClByb3BlcnRpZXMB/4YAAQVDYWNoZQH/iAABBUFsaWFzAQwAAQ5MZWFkaW5nRGlhbW9uZAEMAAEPVHJhaWxpbmdEaWFtb25kAQwAAQhUZW1wbGF0ZQEMAAEKRm9yZWdyb3VuZAEMAAEOVGVtcGxhdGVzTG9naWMBDAABD1Bvd2VybGluZVN5bWJvbAEMAAEKQmFja2dyb3VuZAEMAAEGRmlsbGVyAQwAAQRUeXBlAQwAAQVTdHlsZQEMAAEWTGVhZGluZ1Bvd2VybGluZVN5bWJvbAEMAAELUGxhY2Vob2xkZXIBDAABBFRpcHMB/4oAARNCYWNrZ3JvdW5kVGVtcGxhdGVzAf+MAAEJVGVtcGxhdGVzAf+MAAEORXhjbHVkZUZvbGRlcnMB/4oAAQ5JbmNsdWRlRm9sZGVycwH/igABBU5lZWRzAf+KAAETRm9yZWdyb3VuZFRlbXBsYXRlcwH/jAABBUluZGV4AQQAAQhNaW5XaWR0aAEEAAEIRHVyYXRpb24BBAABCk5hbWVMZW5ndGgBBAABCE1heFdpZHRoAQQAAQdUaW1lb3V0AQQAAQdOZXdsaW5lAQIAAQdFbmFibGVkAQIAAQ9JbnZlcnRQb3dlcmxpbmUBAgABBUZvcmNlAQIAAQdUb2dnbGVkAQIAAQdQZW5kaW5nAQIAAQtJbnRlcmFjdGl2ZQECAAAAE/+FBAEBA01hcAH/hgABDAEQAAAt/4cDAQEFQ2FjaGUB/4gAAQIBCER1cmF0aW9uAQwAAQhTdHJhdGVneQEMAAAAFv+JAgEBCFtdc3RyaW5nAf+KAAEMAAAS/4sCAQEETGlzdAH/jAABDAAAJ/+NBAEBF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9Af+OAAEMARAAAC3/jwMBAQhQYWxldHRlcwH/kAABAgEETGlzdAH/kgABCFRlbXBsYXRlAQwAAAAp/5EEAQEYbWFwW3N0cmluZ11jb2xvci5QYWxldHRlAf+SAAEMAf+CAAA//5MDAQEGQ29uZmlnAf+UAAEDAQhVc2VyTmFtZQH/lgABCEhvc3ROYW1lAf+WAAEJU2hlbGxOYW1lAf+WAAAAE/+VBAEBA01hcAH/lgABDAEMAABc/5cDAQEGQ29uZmlnAf+YAAEGAQZTb3VyY2UBDAABCEludGVydmFsAQwAAQZMYXRlc3QBDAABBEF1dG8BAgABDURpc3BsYXlOb3RpY2UBAgABBUZvcmNlAQIAAAAe/50CAQEPW10qY29uZmlnLkJsb2NrAf+eAAH/mgAA/5D/mQMBAv+aAAEKAQRUeXBlAQwAAQlBbGlnbm1lbnQBDAABBkZpbGxlcgEMAAEIT3ZlcmZsb3cBDAABDkxlYWRpbmdEaWFtb25kAQwAAQ9UcmFpbGluZ0RpYW1vbmQBDAABCFNlZ21lbnRzAf+cAAEHTmV3bGluZQECAAEFRm9yY2UBAgABBUluZGV4AQQAAAAg/5sCAQERW10qY29uZmlnLlNlZ21lbnQB/5wAAf+EAAAU/6ECAQEFQ3ljbGUB/6IAAf+gAAAq/58DAQL/oAABAgEKQmFja2dyb3VuZAEMAAEKRm9yZWdyb3VuZAEMAAAAG/+jAgEBDUlUZXJtRmVhdHVyZXMB/6QAAQwAAP4I/f+AAQcFd2hpdGUHI0UwREVGNAZ5ZWxsb3cHI0YzQUUzNQVibGFjawcjMjYyQjQ0BGJsdWUHIzRCOTVFOQVncmVlbgcjNTlDOUE1Bm9yYW5nZQcjRjA3NjIzA3JlZAcjRDgxRTVCBQdKPHA6eWVsbG93LHRyYW5zcGFyZW50Pu6CtjwvPjwscDp5ZWxsb3c+ID4gPC8+PHA6eWVsbG93LHRyYW5zcGFyZW50Pu6CsDwvPiABB3A6YmxhY2sDC3RyYW5zcGFyZW50AAEHHzxwOnllbGxvdz7uqoMge3sgLlBXRCB9feKdryA8Lz4BB3A6YmxhY2sDC3RyYW5zcGFyZW50AAMBA2NkbgEEMTY4aAADHXt7IC5TaGVsbCB9fSBpbiB7eyAuRm9sZGVyIH19Ai4vVXNlcnMvZGFkYS8uY2FjaGUvb2gtbXktcG9zaC9teXRoZW1lLm9tcC5qc29uAQRqc29uAwIBBnByb21wdAEEbGVmdAUFBQPugrYBA+6CsAEyIHt7IGlmIC5TU0hTZXNzaW9uIH197q6pIHt7IGVuZCB9fXt7IC5Vc2VyTmFtZSB9fSABB3A6YmxhY2sDCHA6eWVsbG93AgdzZXNzaW9uAQdkaWFtb25kAAEBBXN0eWxlBnN0cmluZwwIAAZmb2xkZXIGICDuqoMge3sgcGF0aCAuUGF0aCAuTG9jYXRpb24gfX0gAQdwOndoaXRlAgPugrABCHA6b3JhbmdlAgRwYXRoAQlwb3dlcmxpbmUAAQMTZmV0Y2hfdXBzdHJlYW1faWNvbgRib29sAgIAAQ9icmFuY2hfdGVtcGxhdGUGc3RyaW5nDBgAFnt7IHRydW5jIDI1IC5CcmFuY2ggfX0MZmV0Y2hfc3RhdHVzBGJvb2wCAgABBv/6IHt7IGlmIC5VcHN0cmVhbVVSTCB9fXt7IHVybCAuVXBzdHJlYW1JY29uIC5VcHN0cmVhbVVSTCB9fSB7eyBlbmQgfX17eyAuSEVBRCB9fXt7aWYgLkJyYW5jaFN0YXR1cyB9fSB7eyAuQnJhbmNoU3RhdHVzIH19e3sgZW5kIH19e3sgaWYgLldvcmtpbmcuQ2hhbmdlZCB9fSDvgYQge3sgLldvcmtpbmcuU3RyaW5nIH19e3sgZW5kIH19e3sgaWYgLlN0YWdpbmcuQ2hhbmdlZCB9fSDvgYYge3sgLlN0YWdpbmcuU3RyaW5nIH19e3sgZW5kIH19IAEHcDpibGFjawID7oKwAQdwOmdyZWVuAgNnaXQBCXBvd2VybGluZQQEQnt7IGlmIG9yICguV29ya2luZy5DaGFuZ2VkKSAoLlN0YWdpbmcuQ2hhbmdlZCkgfX1wOnllbGxvd3t7IGVuZCB9fTd7eyBpZiBhbmQgKGd0IC5BaGVhZCAwKSAoZ3QgLkJlaGluZCAwKSB9fXA6cmVke3sgZW5kIH19JHt7IGlmIGd0IC5BaGVhZCAwIH19IzQ5NDE2RHt7IGVuZCB9fSV7eyBpZiBndCAuQmVoaW5kIDAgfX0jN0EzMDZDe3sgZW5kIH19BQNBe3sgaWYgb3IgKC5Xb3JraW5nLkNoYW5nZWQpICguU3RhZ2luZy5DaGFuZ2VkKSB9fXA6YmxhY2t7eyBlbmQgfX05e3sgaWYgYW5kIChndCAuQWhlYWQgMCkgKGd0IC5CZWhpbmQgMCkgfX1wOndoaXRle3sgZW5kIH19JHt7IGlmIGd0IC5BaGVhZCAwIH19cDp3aGl0ZXt7IGVuZCB9fQAHBSDvg6cgAQdwOndoaXRlAgPugrABCHA6eWVsbG93AgRyb290AQlwb3dlcmxpbmUAAQEOYWx3YXlzX2VuYWJsZWQEYm9vbAICAAEEHjx0cmFuc3BhcmVudCxiYWNrZ3JvdW5kPu6CsDwvPgED7oK0AS4ge3sgaWYgZ3QgLkNvZGUgMCB9fe+AjXt7IGVsc2UgfX3vgIx7eyBlbmQgfX0gAQdwOndoaXRlAwZwOmJsdWUCBnN0YXR1cwEHZGlhbW9uZAQBIXt7IGlmIGd0IC5Db2RlIDAgfX1wOnJlZHt7IGVuZCB9fQAAAQdycHJvbXB0BgUBAwxkaXNwbGF5X21vZGUGc3RyaW5nDAcABWZpbGVzFWZldGNoX3BhY2thZ2VfbWFuYWdlcgRib29sAgIAAAxob21lX2VuYWJsZWQEYm9vbAICAAAGBO6cmCABB3A6Z3JlZW4DC3RyYW5zcGFyZW50AgRub2RlAQVwbGFpbgABAQ1mZXRjaF92ZXJzaW9uBGJvb2wCAgAABgTumKYgAQZwOmJsdWUDC3RyYW5zcGFyZW50AgJnbwEFcGxhaW4AAQMMZGlzcGxheV9tb2RlBnN0cmluZwwHAAVmaWxlcw1mZXRjaF92ZXJzaW9uBGJvb2wCAgAAEWZldGNoX3ZpcnR1YWxfZW52BGJvb2wCAgAABgTuiLUgAQhwOnllbGxvdwMLdHJhbnNwYXJlbnQCBnB5dGhvbgEFcGxhaW4AByFpbiA8cDpibHVlPjxiPnt7IC5OYW1lIH19PC9iPjwvPiABB3A6d2hpdGUDC3RyYW5zcGFyZW50AgVzaGVsbAEFcGxhaW4ABzlhdCA8cDpibHVlPjxiPnt7IC5DdXJyZW50RGF0ZSB8IGRhdGUgIjE1OjA0OjA1IiB9fTwvYj48Lz4BB3A6d2hpdGUDC3RyYW5zcGFyZW50AgR0aW1lAQVwbGFpbgAAAwIBAQ9kaXNwbGF5X2RlZmF1bHQEYm9vbAICAAEEA+6CsAED7oK0ATsg7p6tIHt7IC5Qcm9maWxlIH19e3sgaWYgLlJlZ2lvbiB9fUB7eyAuUmVnaW9uIH19e3sgZW5kIH19IAEHcDp3aGl0ZQMIcDpvcmFuZ2UCA2F3cwEHZGlhbW9uZAMBA2F3cwABAQ9kaXNwbGF5X2RlZmF1bHQEYm9vbAICAAEEA+6CsAED7oK0AREg7q+YIHt7IC5OYW1lIH19IAEHcDp3aGl0ZQMGcDpibHVlAgJhegEHZGlhbW9uZAMBAmF6AAEIAgECAQUUAA==
[TRACE] 18:48:02.809 store.go:](session CONFIG) - 253.417µs
[TRACE] 18:48:02.809 gob.go:Restore() - 74.083µs
[TRACE] 18:48:02.809 gob.go:Get() - 333.334µs
[TRACE] 18:48:02.809 terminal.go:Shell() - 42ns
[TRACE] 18:48:02.809 terminal.go:Flags() - 41ns
[DEBUG] 18:48:02.809 store.go:]:154 → (session) found entry: template_cache - {map[Path:0x53a7df268480 Session:0x53a7df45de00 Shell:0x53a7df400b90 Status:0x53a7df400af0 Time:0x53a7df400c80] map[] ~ dada  dada XMC-MBP 5.9 zsh /Users/dada darwin 29.6.1 2 1 0 2 false false}
[TRACE] 18:48:02.809 store.go:](session template_cache) - 17.417µs
[TRACE] 18:48:02.809 cache.go:restoreCache() - 20.541µs
[DEBUG] 18:48:02.809 writer.go:Init:107 → terminal program: ghostty
[DEBUG] 18:48:02.809 writer.go:Init:108 → terminal shell: shell
[DEBUG] 18:48:02.809 terminal.go:Getenv:64 → NO DATA
[TRACE] 18:48:02.809 terminal.go:Getenv(OMP_CACHE_DISABLED) - 2.083µs
[DEBUG] 18:48:02.809 store.go:]:141 → (device) key not found: accent_color
[TRACE] 18:48:02.809 store.go:](device accent_color) - 5.084µs
[ERROR] 18:48:02.816 terminal.go:RunCommand:277 → exit status 1
[DEBUG] 18:48:02.816 terminal.go:RunCommand:280 ↓
    2026-03-02 18:48:02.816 defaults[29619:30569893]
    The domain/default pair of (kCFPreferencesAnyApplication, AppleAccentColor) does not exist
[TRACE] 18:48:02.816 terminal.go:RunCommand(defaults read -g AppleAccentColor) - 7.001791ms
[ERROR] 18:48:02.816 colors_darwin.go:GetAccentColor:14 → exit status 1
[TRACE] 18:48:02.816 colors.go:SetAccentColor() - 7.02ms
[TRACE] 18:48:02.816 terminal.go:Shell() - 84ns
[DEBUG] 18:48:02.816 terminal.go:Getenv:64 → 5.9
[TRACE] 18:48:02.816 terminal.go:Getenv(POSH_SHELL_VERSION) - 3.667µs
[DEBUG] 18:48:02.816 debug.go:PrintDebug:24 → segment: Title
[DEBUG] 18:48:02.816 text.go:patchTemplate:149 → {{ .Shell }} in {{ .Folder }}
[TRACE] 18:48:02.817 terminal.go:Flags() - 42ns
[DEBUG] 18:48:02.817 terminal.go:Getenv:64 → NO DATA
[TRACE] 18:48:02.817 terminal.go:Getenv(POSH_CURSOR_LINE) - 1.791µs
[DEBUG] 18:48:02.817 terminal.go:Getenv:64 → NO DATA
[TRACE] 18:48:02.817 terminal.go:Getenv(POSH_CURSOR_COLUMN) - 5.709µs
[TRACE] 18:48:02.817 terminal.go:Flags() - 0s
[TRACE] 18:48:02.817 terminal.go:Flags() - 42ns
[TRACE] 18:48:02.817 terminal.go:Flags() - 42ns
[TRACE] 18:48:02.817 terminal.go:Flags() - 41ns
[TRACE] 18:48:02.817 terminal.go:Flags() - 41ns
[TRACE] 18:48:02.817 terminal.go:Flags() - 42ns
[TRACE] 18:48:02.817 terminal.go:Flags() - 41ns
[TRACE] 18:48:02.817 terminal.go:Flags() - 0s
[TRACE] 18:48:02.817 terminal.go:Flags() - 42ns
[TRACE] 18:48:02.817 terminal.go:Flags() - 42ns
[TRACE] 18:48:02.817 terminal.go:Flags() - 84ns
[DEBUG] 18:48:02.817 home.go:12 → /Users/dada
[DEBUG] 18:48:02.817 home.go:12 → /Users/dada
[DEBUG] 18:48:02.817 home.go:12 → /Users/dada
[DEBUG] 18:48:02.817 home.go:12 → /Users/dada
[TRACE] 18:48:02.817 terminal.go:GOOS() - 125ns
[TRACE] 18:48:02.817 terminal.go:GOOS() - 42ns
[TRACE] 18:48:02.817 terminal.go:GOOS() - 84ns
[TRACE] 18:48:02.817 terminal.go:GOOS() - 42ns
[DEBUG] 18:48:02.817 segment.go:Execute:173 → segment: Status
[DEBUG] 18:48:02.817 segment.go:Execute:173 → segment: Root
[DEBUG] 18:48:02.817 segment.go:Execute:173 → segment: Path
[DEBUG] 18:48:02.817 segment.go:Execute:173 → segment: Git
[DEBUG] 18:48:02.817 store.go:]:154 → (session) found entry: toggle_cache - map[]
[DEBUG] 18:48:02.817 store.go:]:154 → (session) found entry: toggle_cache - map[]
[DEBUG] 18:48:02.817 store.go:]:154 → (session) found entry: toggle_cache - map[]
[DEBUG] 18:48:02.817 store.go:]:154 → (session) found entry: toggle_cache - map[]
[TRACE] 18:48:02.817 store.go:](session toggle_cache) - 6.541µs
[DEBUG] 18:48:02.817 segment.go:isToggled:291 → no toggles found
[TRACE] 18:48:02.817 store.go:](session toggle_cache) - 7.958µs
[DEBUG] 18:48:02.817 segment.go:isToggled:291 → no toggles found
[DEBUG] 18:48:02.817 segment.go:isToggled:291 → no toggles found
[DEBUG] 18:48:02.817 segment.go:isToggled:291 → no toggles found
[TRACE] 18:48:02.817 terminal.go:StatusCodes() - 125ns
[TRACE] 18:48:02.817 terminal_unix.go:Root() - 458ns
[DEBUG] 18:48:02.817 map.go:String:79 → status_template: {{ .Code }}
[TRACE] 18:48:02.817 terminal.go:Flags() - 0s
[DEBUG] 18:48:02.817 map.go:Bool:133 → display_cygpath: false
[TRACE] 18:48:02.817 terminal.go:GOOS() - 0s
[TRACE] 18:48:02.817 separator.go:Separator() - 41ns
[TRACE] 18:48:02.817 terminal.go:Shell() - 42ns
[TRACE] 18:48:02.817 terminal.go:GOOS() - 41ns
[DEBUG] 18:48:02.817 map.go:Bool:133 → mapped_locations_enabled: true
[DEBUG] 18:48:02.817 map.go:String:79 → windows_registry_icon: 
[DEBUG] 18:48:02.817 map.go:String:79 → windows_registry_icon: 
[DEBUG] 18:48:02.817 home.go:12 → /Users/dada
[TRACE] 18:48:02.817 separator.go:Separator() - 41ns
[TRACE] 18:48:02.817 terminal.go:Flags() - 84ns
[DEBUG] 18:48:02.817 home.go:12 → /Users/dada
[TRACE] 18:48:02.817 terminal.go:GOOS() - 167ns
[DEBUG] 18:48:02.817 segment.go:Execute:173 → segment: Session
[DEBUG] 18:48:02.817 text.go:patchTemplate:149 → {{ .Code }}
[DEBUG] 18:48:02.817 store.go:]:154 → (session) found entry: toggle_cache - map[]
[TRACE] 18:48:02.817 terminal.go:GOOS() - 42ns
[TRACE] 18:48:02.817 terminal.go:GOOS() - 42ns
[TRACE] 18:48:02.817 store.go:](session toggle_cache) - 9.125µs
[TRACE] 18:48:02.817 terminal.go:Flags() - 0s
[DEBUG] 18:48:02.817 map.go:String:79 → home_icon: ~
[DEBUG] 18:48:02.817 segment.go:isToggled:291 → no toggles found
[DEBUG] 18:48:02.817 map.go:KeyValueMap:208 → mapped_locations: map[]
[DEBUG] 18:48:02.817 terminal.go:Getenv:64 → NO DATA
[TRACE] 18:48:02.817 separator.go:Separator() - 41ns
[TRACE] 18:48:02.817 terminal.go:Getenv(SSH_CONNECTION) - 2.792µs
[TRACE] 18:48:02.817 terminal.go:GOOS() - 42ns
[TRACE] 18:48:02.817 terminal.go:GOOS() - 41ns
[TRACE] 18:48:02.817 separator.go:Separator() - 42ns
[DEBUG] 18:48:02.817 store.go:]:154 → (device) found entry: environment_platform - darwin
[TRACE] 18:48:02.817 terminal.go:GOOS() - 0s
[TRACE] 18:48:02.817 terminal.go:GOOS() - 42ns
[TRACE] 18:48:02.817 store.go:](device environment_platform) - 3.667µs
[TRACE] 18:48:02.817 terminal.go:GOOS() - 0s
[TRACE] 18:48:02.817 terminal.go:GOOS() - 42ns
[TRACE] 18:48:02.817 terminal.go:GOOS() - 42ns
[TRACE] 18:48:02.817 terminal.go:Flags() - 42ns
[DEBUG] 18:48:02.817 map.go:StringArray:220 → cycle: []
[DEBUG] 18:48:02.817 map.go:String:79 → folder_separator_template:
[DEBUG] 18:48:02.817 map.go:String:79 → folder_separator_icon: /
[DEBUG] 18:48:02.817 map.go:Bool:133 → cycle_folder_separator: false
[DEBUG] 18:48:02.817 map.go:String:79 → folder_format: %s
[DEBUG] 18:48:02.817 map.go:String:79 → edge_format: %s
[DEBUG] 18:48:02.817 map.go:String:79 → left_format: %s
[DEBUG] 18:48:02.817 map.go:String:79 → right_format: %s
[TRACE] 18:48:02.817 terminal.go:Flags() - 42ns
[TRACE] 18:48:02.817 terminal.go:GOOS() - 0s
[TRACE] 18:48:02.817 terminal.go:StackCount() - 42ns
[TRACE] 18:48:02.817 terminal_unix.go:DirIsWritable(/Users/dada) - 17.75µs
[TRACE] 18:48:02.817 terminal.go:Flags() - 0s
[DEBUG] 18:48:02.817 map.go:Bool:133 → disable_with_jj: false
[DEBUG] 18:48:02.817 terminal.go:ResolveSymlink:195 → /Users/dada
[TRACE] 18:48:02.817 terminal.go:ResolveSymlink(/Users/dada) - 15.666µs
[ERROR] 18:48:02.817 terminal.go:HasParentFilePath:464 → stat .git: no such file or directory
[TRACE] 18:48:02.817 terminal.go:HasParentFilePath(.git) - 38.042µs
[TRACE] 18:48:02.817 terminal.go:Flags() - 42ns
[DEBUG] 18:48:02.821 terminal.go:RunCommand:280 → dada                          Mar  2 18:48
[TRACE] 18:48:02.821 terminal.go:RunCommand(who am i) - 3.805ms
[TRACE] 18:48:02.821 terminal.go:Flags() - 42ns
[DEBUG] 18:48:02.821 text.go:patchTemplate:149 →  {{ if .Data.SSHSession }} {{ end }}{{ .UserName }}
[DEBUG] 18:48:02.822 text.go:patchTemplate:149 →   {{ path .Data.Path .Data.Location }}
[DEBUG] 18:48:02.822 text.go:patchTemplate:149 →  {{ if gt .Code 0 }}{{ else }}{{ end }}
[DEBUG] 18:48:02.822 text.go:patchTemplate:149 → {{ if gt .Code 0 }}p:red{{ end }}
[DEBUG] 18:48:02.822 text.go:patchTemplate:149 → {{ if gt .Code 0 }}p:red{{ end }}
[TRACE] 18:48:02.822 terminal.go:Shell() - 167ns
[TRACE] 18:48:02.822 terminal.go:Flags() - 0s
[TRACE] 18:48:02.822 terminal.go:Flags() - 42ns
[TRACE] 18:48:02.822 terminal.go:Flags() - 0s
[TRACE] 18:48:02.822 terminal.go:Flags() - 0s
[TRACE] 18:48:02.822 terminal.go:Flags() - 0s
[TRACE] 18:48:02.822 terminal.go:Flags() - 42ns
[TRACE] 18:48:02.822 terminal.go:Flags() - 42ns
[DEBUG] 18:48:02.822 home.go:12 → /Users/dada
[TRACE] 18:48:02.822 terminal.go:GOOS() - 84ns
[DEBUG] 18:48:02.822 home.go:12 → /Users/dada
[DEBUG] 18:48:02.822 segment.go:Execute:173 → segment: Node
[TRACE] 18:48:02.822 terminal.go:Flags() - 42ns
[TRACE] 18:48:02.822 terminal.go:GOOS() - 42ns
[DEBUG] 18:48:02.822 segment.go:Execute:173 → segment: Time
[DEBUG] 18:48:02.822 store.go:]:154 → (session) found entry: toggle_cache - map[]
[DEBUG] 18:48:02.822 home.go:12 → /Users/dada
[TRACE] 18:48:02.822 terminal.go:Flags() - 42ns
[DEBUG] 18:48:02.822 store.go:]:154 → (session) found entry: toggle_cache - map[]
[TRACE] 18:48:02.822 terminal.go:GOOS() - 0s
[TRACE] 18:48:02.822 store.go:](session toggle_cache) - 4.583µs
[TRACE] 18:48:02.822 store.go:](session toggle_cache) - 3µs
[DEBUG] 18:48:02.822 segment.go:isToggled:291 → no toggles found
[DEBUG] 18:48:02.822 segment.go:Execute:173 → segment: Shell
[DEBUG] 18:48:02.822 segment.go:isToggled:291 → no toggles found
[DEBUG] 18:48:02.822 home.go:12 → /Users/dada
[TRACE] 18:48:02.822 terminal.go:GOOS() - 0s
[DEBUG] 18:48:02.822 store.go:]:154 → (session) found entry: toggle_cache - map[]
[DEBUG] 18:48:02.822 segment.go:Execute:173 → segment: Python
[DEBUG] 18:48:02.822 map.go:String:79 → time_format: 15:04:05
[TRACE] 18:48:02.822 store.go:](session toggle_cache) - 3.791µs
[DEBUG] 18:48:02.822 store.go:]:154 → (session) found entry: togg[TRACE] 18:48:02.822 terminal.go:Flags() - 41ns
[DEBUG] 18:48:02.822 map.go:KeyValueMap:208 → mapped_shell_names: map[]
[TRACE] 18:48:02.822 terminal.go:Flags() - 0s
[TRACE] 18:48:02.822 store.go:](session toggle_cache) - 3.417µs
[TRACE] 18:48:02.822 terminal.go:Shell() - 42ns
[DEBUG] 18:48:02.822 segment.go:isToggled:291 → no toggles found
[TRACE] 18:48:02.822 terminal.go:Flags() - 0s
[DEBUG] 18:48:02.822 home.go:12 → /Users/dada
[TRACE] 18:48:02.822 terminal.go:GOOS() - 0s
[TRACE] 18:48:02.822 terminal.go:Flags() - 0s
[DEBUG] 18:48:02.822 segment.go:Execute:173 → segment: Go
[DEBUG] 18:48:02.822 map.go:String:83 → display_mode: files
[DEBUG] 18:48:02.822 map.go:StringArray:220 → extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .vue]
[DEBUG] 18:48:02.822 store.go:]:154 → (session) found entry: toggle_cache - map[]
[DEBUG] 18:48:02.822 map.go:StringArray:220 → folders: []
[DEBUG] 18:48:02.822 map.go:StringArray:220 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[DEBUG] 18:48:02.822 map.go:Bool:141 → home_enabled: false
[DEBUG] 18:48:02.822 segment.go:isToggled:291 → no toggles found
[DEBUG] 18:48:02.822 home.go:12 → /Users/dada
[DEBUG] 18:48:02.822 map.go:StringArray:220 → folders: [.venv venv virtualenv venv-win pyenv-win]
[DEBUG] 18:48:02.822 map.go:Bool:133 → home_enabled: false
[TRACE] 18:48:02.822 terminal.go:Flags() - 41ns
[DEBUG] 18:48:02.822 home.go:12 → /Users/dada
[TRACE] 18:48:02.822 terminal.go:Flags() - 0s
[DEBUG] 18:48:02.822 map.go:StringArray:220 → extensions: [*.go go.mod go.sum go.work go.work.sum]
[DEBUG] 18:48:02.822 map.go:StringArray:220 → folders: []
[DEBUG] 18:48:02.822 map.go:Bool:133 → home_enabled: false
[DEBUG] 18:48:02.822 home.go:12 → /Users/dada
[TRACE] 18:48:02.822 terminal.go:Flags() - 42ns
[DEBUG] 18:48:02.822 text.go:patchTemplate:149 → in <p:blue><b>{{ .Data.Name }}</b></>
[DEBUG] 18:48:02.822 text.go:patchTemplate:149 → at <p:blue><b>{{ .Data.CurrentDate | date "15:04:05" }}</b></>
[TRACE] 18:48:02.822 terminal.go:Shell() - 42ns
[TRACE] 18:48:02.822 terminal.go:Flags() - 42ns
[TRACE] 18:48:02.822 terminal.go:Shell() - 0s
[DEBUG] 18:48:02.825 terminal_unix.go:TerminalWidth:83 → terminal width: 99
[TRACE] 18:48:02.825 terminal_unix.go:TerminalWidth() - 3.036292ms
[TRACE] 18:48:02.825 path.go:Path() - 42ns
```

--- a/src/shell/scripts/omp.zsh
+++ b/src/shell/scripts/omp.zsh
@@ -83,7 +83,8 @@ function _omp_start_streaming() {
 
   # start process substitution — no PID tracking needed
   # closing fd sends SIGPIPE which terminates oh-my-posh
-  exec {_omp_stream_fd}< <(exec "${stream_cmd[@]}") 2>/dev/null
+  # redirect stream process stdin to prevent it from interfering with command output
+  exec {_omp_stream_fd}< <(exec "${stream_cmd[@]}" </dev/null 2>/dev/null)
 
   if [[ $_omp_stream_fd -lt 0 ]]; then
     return 1
@@ -146,7 +147,7 @@ function _omp_precmd() {
   _omp_no_status=true
   _omp_tooltip_command=''
 
-  if [ $_omp_start_time ]; then
+  if [[ -n $_omp_start_time ]]; then
     local omp_now=$($_omp_executable get millis)
     _omp_execution_time=$(($omp_now - $_omp_start_time))
     _omp_no_status=false

